test/python/common.py

```diff
@@ -289,5 +298,200 @@ def _is_ci_fork_pull_request():
     return False
 
 
-SKIP_ONLINE_TESTS = os.getenv('SKIP_ONLINE_TESTS', _is_ci_fork_pull_request())
 SKIP_SLOW_TESTS = os.getenv('SKIP_SLOW_TESTS', True) not in ['false', 'False', '-1']
+RECORD_TEST_RESPONSE = os.getenv('RECORD_TEST_RESPONSE', False) is not False
+VCR_MODE = 'none'
+if RECORD_TEST_RESPONSE:
+    SKIP_SLOW_TESTS = True  # TODO Activate later
+    VCR_MODE = 'all'
+
+
+class IdRemoverPersister(FilesystemPersister):
```

Comment:
Not a full review yet - but can you move this class and the VCR related functionality (except the bits you might need in the decorator, etc) to a new file, in the hopes of keeping `common.py` at a manageable size and complexity?

Reply:
Addressed in b3215c8